src/ripple/net/ShardDownloader.md

```diff
@@ -177,7 +177,7 @@ The `download_path` field of the `shard_db` entry is used to determine where to
 type=NuDB
 path=/var/lib/rippled/db/shards/nudb
 download_path=/var/lib/rippled/db/shards/
-max_size_gb=50
+max_historical_shards=50
```

Comment:
Yes, the `max_size_gb` key is being removed. The decision came from @miguelportilla. This is a preexisting convention BTW, since the node store limits are determined by ledger counts and not drive capacity.

Reply:
I defer to you and @miguelportilla on this. It's probably OK to break backward compatibility as few people are using shards right now.